File: php/samples/SimpleOrm.Sample/src/Migrations/Table/Transaction/V0003_AddNote.php
<?php

declare(strict_types=1);

namespace SimpleOrm\Sample\Migrations\Table\Transaction;

use SimpleOrm\Migrations\TableActions;
use SimpleOrm\Migrations\TableMigration;
use SimpleOrm\Sample\Models\Tables\Transaction;

final class V0003_AddNote extends TableMigration
{
    public function entityClass(): string
    {
        return Transaction::class;
    }

    public function action(TableActions $actions): void
    {
        $actions->addColumn('note', 'TEXT');
    }
}

File: go/orm/sample/migrations/Table/Transaction/V0003_AddNote.go
package transaction

import (
	"github.com/kosmassp/SimpleOrm/go/orm"
	"github.com/kosmassp/SimpleOrm/go/orm/sample"
)

// V0003_AddNote adds an optional free-text note.
type V0003_AddNote struct {
	orm.TableMigration[sample.Transaction]
}

func (V0003_AddNote) Action(actions *orm.TableActions) {
	actions.AddColumn("note", "TEXT")
}
